src/index.js

```diff
@@ -55,7 +56,7 @@ export default class extends Component {
     this.editor = unlayer.createEditor({
       ...options,
       id: this.editorId,
-      displayMode: 'email',
+      displayMode,
```

Comment:
maybe this line should come before the `...options,` line, so we they can put all config inside the `options` object instead of being required to use the `displayMode` prop. 

or this: `displayMode: displayMode || options?.displayMode`.

@lucasbesen

Reply:
@lucasbesen @adeelraza ^

Reply:
pushed a commit with the second option above.